gh-actions/common/build-debian/prepare-container.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -eux
+
+# Disable installing of manpages and docs
+cat <<"EOF" | tee /etc/dpkg/dpkg.cfg.d/01_nodoc
+# Delete man pages
+path-exclude=/usr/share/man/*
+
+# Delete docs
+path-exclude=/usr/share/doc/*
+path-include=/usr/share/doc/*/copyright
+EOF
+
+echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90aptyes
+apt update
+
+locales_required_check=$(dirname "$0")/.locales-required
+if ! [ -e "${locales_required_check}" ] && [ -f ./debian/control ]; then
+  apt install dctrl-tools
```

Comment:
didn’t we say we always use `-y`?

Reply:
Well yeah... I had avoided it through env variables though, but let me do it...

Reply:
Actually it would be pretty dumb, because exactly to avoid this to be forgotten I did initially:

`echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90aptyes`

Reply:
You’re correct